packages/static-module-record/src/babelPlugin.js

```diff
@@ -253,31 +239,33 @@ function makeModulePlugins(options) {
 
       // Create the export calls.
       const isConst = decl.kind === 'const';
-      const replacements = rewriteVars(
+      const replace = rewriteVars(
```

Comment:
I did like your rename from replace to replacements, which looks like it’s caught in a revert here.